packages/visual-editor/src/components/Dropzone/useComponent.ts

```diff
@@ -67,30 +69,37 @@ export const useComponent = ({
     areEntitiesFetched,
     resolveDesignValue,
     renderDropzone,
-    definition: componentRegistration.definition,
+    definition: componentRegistration?.definition,
     userIsDragging,
   });
 
+  const createElementToRender = (componentRegistration: ComponentRegistration) => {
```

Comment:
We definitely have some changes to this same part of the code incoming, so there might be some MCs to work through. Should be fine though.